doc/developers/advanced_installation.rst

```diff
@@ -63,9 +63,17 @@ feature, code or documentation improvement).
 
    .. prompt:: bash $
 
-     conda create -n sklearn-env -c conda-forge python=3.9 numpy scipy cython
+     conda env create -f environments/common.yml
      conda activate sklearn-env
 
+    .. note::
+        The default name for the environment is `sklearn-env` however you may
+        customize this by setting the `-n` flag to whatever your desired name
+        is. For example to name your environment `sklearn-custom`, you would run
+        `conda env create -f environments/common.yml -n sklearn-custom`. Note that
+        doing this requires you to carry on the `-n <env_name>` flag to any potential
+        `conda env update` commands when installing additional dependencies.
```

Comment:
I would reduce the note by stating what environment name will be given and then a link to the conda documentation to go more depth on managing the environment: https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment